turbopack/crates/turbo-tasks-backend/benches/overhead.rs

```diff
@@ -125,14 +190,26 @@ fn run_turbo<Mode: TurboMode>(
                 // If cached run once outside the loop to ensure the tasks are cached.
                 if Mode::is_cached() {
                     for i in 0..iters {
-                        black_box(busy_turbo(i, black_box(d)).await?);
+                        black_box(busy_turbo(Mode::key(i), black_box(d)).await?);
```

Comment:
i actualyl didn't want to do this. to ensure we were reading, even in the single key case, from a 'large' cache.  So the only difference between those two scenarios was the 'scatter reads'